tests/NPSP.py

```diff
@@ -749,15 +813,94 @@ def verify_payment_details(self):
     #             self.selenium.page_should_contain_element(loctor_contains)
 
     def click_object_manager_button(self,title):  
-        """clicks on the + button next to contact on manage hh page"""      
+        """clicks on the buttons in object manager"""      
         locator=npsp_lex_locators['object_manager']['button'].format(title)
         self.selenium.get_webelement(locator).click()  
-
+        
+    def click_bge_button(self,text):  
+        """clicks on buttons for BGE"""      
+        locator=npsp_lex_locators['bge']['button'].format(text)
+        #self.selenium.get_webelement(locator).click()
+        self.selenium.click_button(locator)     
+    
+    def verify_title(self,title,value):
+        """"""
+        locator=npsp_lex_locators['bge']['title'].format(title,value)
+        actual_value=self.selenium.get_webelement(locator).text
+        assert value == actual_value, "Expected value to be {} but found {}".format(
+            value, actual_value
+        )  
+#     def get_npsp_record_id(self):
+#         """ Parses the current url to get the object id of the current record.
+#             Expects url format like: [a-zA-Z0-9]{15,18}
+#         """
+#         url = self.selenium.get_location()
+#         for part in url.split("/"):
+#             if re.match(r"^[a-zA-Z0-9]{15,18}$", part):
+#                 return part
+#         raise AssertionError("Could not parse record id from url: {}".format(url))            
```

Comment:
@skristem could you delete this instead of commenting out? We can always find deleted code in the git history by looking in the code changes for the PR that deleted it, so I think it helps keep things clear to remove code that is no longer used.

Reply:
done